agent/cmd/cuda-checkpoint-helper/main.cpp

```
@@ -0,0 +1,1561 @@
+    output_restore_failed =
+        dup2(saved_stderr, STDERR_FILENO) < 0 || output_restore_failed;
+    close(saved_stderr);
+  }
+  if (output_restore_failed) {
+    response.cuda_status = CUDA_ERROR_OPERATING_SYSTEM;
+    response.flags |= daemon_protocol::kResponseFatal;
+  }
+  bool output_truncated = false;
+  bool error_truncated = false;
+  std::string capture_error;
+  if (!output_capture.Finish(&response.output, &output_truncated,
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)